pkg/supportbundle/supportbundle.go

```diff
@@ -226,3 +226,24 @@ func AnalyzeSupportBundle(spec *troubleshootv1beta2.SupportBundleSpec, tmpDir st
 	}
 	return analyzeResults, nil
 }
+
+// the intention with these appends is to swap them out at a later date with more specific handlers for merging the spec fields
+func ConcatSpec(target *troubleshootv1beta2.SupportBundle, source *troubleshootv1beta2.SupportBundle) *troubleshootv1beta2.SupportBundle {
+	newBundle := target.DeepCopy()
+	for _, v := range source.Spec.Collectors {
```

Comment:
No need for any of these loops
```
newBundle.Spec.Collectors = append(target.Spec.Collectors, source.Spec.Collectors...)
```

Reply:
Thanks! I didn't know about this magic variadic argument syntax